client/src/contexts/QA.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useContext } from 'react';
+import axios from 'axios';
+
+export const QAContext = React.createContext();
+
+export const QAProvider({ children }) {
```

Comment:
missing the step of setting the state with the questions/answers after fetching them: lines 7 & 8 can be 
```
const [questions, setQuestions] = useState({});
const [answers, setAnswers] = useState({});
```